engine/crates/runtime/src/cache/mod.rs

```diff
@@ -38,15 +38,16 @@ pub enum EntryState {
 /// Wraps an entry from cache when getting it from there
 #[derive(Debug, PartialEq, Eq)]
 pub enum Entry<T> {
-    Hit(T),
+    /// A hit, and the current time till stale of that hit
```

Comment:
would say current `age`, and rename `max_age` underneath to `age` to follow the example of the `Age` header.

Reply:
It's not the age though?  It's the time till it's dead

Reply:
ah, would just say that in the comment then :D
And use `time_left` or something like that rather than `max_age` underneath

Reply:
> ah, would just say that in the comment then :D

You find "time till dead" clearer than "time till stale"?  Happy to change it if so, but stale seemed the better word to me.

Will fix the `max_age` bit though, yes

Reply:
no looks good actually, guess would have worded it this way, but it's clear.